api/src/main/resources/org/openmrs/api/db/hibernate/Diagnosis.hbm.xml

```diff
@@ -21,13 +21,16 @@
 			<generator class="native" />
 		</id>
 
-		<property name="diagnosis" column="coded_or_free_text" length="50" not-null="true">
-			<type name="org.openmrs.CodedOrFreeText"/>
-		</property>
+		<component name="diagnosis" class="org.openmrs.CodedOrFreeText">
+			<many-to-one name="coded" class="org.openmrs.Concept" column="encounter_diagnosis_coded" />
+			<property name="nonCoded" type="java.lang.String" length="255" column="encounter_diagnosis_non_coded" />
```

Comment:
The column names do not match the ones in the liquibase changeset

Reply:
I have changed them appropriately.

Reply:
Some column names are not correct